apps/frontend/src/libs/components/sidebar/sidebar-item/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.sidebar-item > * {
+	display: flex;
+	flex-direction: row;
+	gap: 8px;
+	align-items: center;
+	padding: 12px 16px;
+	color: var(--color-text-secondary);
+	text-decoration: none;
+}
+
+.sidebar-item > *:hover {
+	color: var(--color-text-primary);
+	background-color: var(--color-background-secondary);
+	border-radius: 6px;
+}
+
+.sidebar-item > *:hover svg {
```

Comment:
Instead of `.sidebar-item > *:hover svg` use a specific selector. Icon color should be automatically taken from the current color, we don't need to access svg inside it. Also as I remember there is a color property for Icon

Reply:
I need to change icon color when hovering over navigation element. I think it is the easiest solution for that, or should i do that using ts?